tests/ui-toml/impl_trait_in_params/impl_trait_in_params.stderr

```diff
@@ -0,0 +1,41 @@
+error: trait `Trait` is more private than the item `Public::t`
```

Comment:
This error seems unrelated to what we want to test here. I guess you have to make `Trait` trait public.